src/Raven.Server/Documents/Commands/Replication/GetReplicationOngoingTasksProgressCommand.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Net.Http;
+using Raven.Client.Http;
+using Raven.Server.Documents.Replication.Stats;
+using Raven.Server.Json;
+using Sparrow.Json;
+
+namespace Raven.Server.Documents.Commands.Replication
+{
+    internal sealed class GetReplicationOngoingTasksProgressCommand : RavenCommand<ReplicationTaskProgress[]>
```

Comment:
why don't we want to expose it to client api ?

Reply:
TBH, I saw that `GetEtlTaskProgressCommand` is on the server, so I thought this should be handled the same way